<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Azael Avalos &lt;coproscefalo@gmail.com&gt;</author>
 <class>GeneralSettings</class>
 <widget class="QWidget" name="GeneralSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" alignment="Qt::AlignLeft">
    <widget class="QCheckBox" name="pointing_device_checkbox">
     <property name="toolTip">
      <string>Toggles the internal pointing device(s) via hardware.</string>
     </property>
     <property name="whatsThis">
      <string>Toggles the internal pointing device(s) via hardware.</string>
     </property>
     <property name="text">
      <string>Internal Pointing device</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignLeft">
    <widget class="QCheckBox" name="built_in_lan_checkbox">
     <property name="toolTip">
      <string>Toggles the built in LAN device.</string>
     </property>
     <property name="whatsThis">
      <string>Toggles the built in LAN device.</string>
     </property>
     <property name="text">
      <string>Built In LAN</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>USB Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0" alignment="Qt::AlignLeft">
       <widget class="QCheckBox" name="rapid_charge_checkbox">
        <property name="toolTip">
         <string>Toggles the USB rapid charge feature.</string>
        </property>
        <property name="whatsThis">
         <string>Toggles the USB rapid charge feature.</string>
        </property>
        <property name="text">
         <string>USB Rapid Charge</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignLeft">
       <widget class="QCheckBox" name="usb_three_checkbox">
        <property name="toolTip">
         <string>Toggles USB 3 functionality, acting as a regular USB 2 when disabled.</string>
        </property>
        <property name="whatsThis">
         <string>Toggles USB 3 functionality, acting as a regular USB 2 when disabled.</string>
        </property>
        <property name="text">
         <string>USB 3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignLeft">
       <widget class="QCheckBox" name="usb_legacy_checkbox">
        <property name="toolTip">
         <string>Toggles the USB Legacy Emulation feature.</string>
        </property>
        <property name="whatsThis">
         <string>Toggles the USB Legacy Emulation feature.</string>
        </property>
        <property name="text">
         <string>USB Legacy Emulation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" alignment="Qt::AlignLeft">
    <widget class="QLabel" name="power_on_display_label">
     <property name="text">
      <string>Power On Display:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="power_on_display_combobox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Select the desired display device to be used at Power On.</string>
     </property>
     <property name="whatsThis">
      <string>Select the desired display device to be used at Power On.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="hdmiCECGroupBox">
     <property name="toolTip">
      <string>Toggles the HDMI-CEC features, HDMI-CEC allows devices connected through HDMI ports to communicate back and forth.</string>
     </property>
     <property name="whatsThis">
      <string>Toggles the HDMI-CEC features, HDMI-CEC allows devices connected through HDMI ports to communicate back and forth.</string>
     </property>
     <property name="title">
      <string>HDMI Consumer Electronics Control</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0" alignment="Qt::AlignLeft">
       <widget class="QCheckBox" name="hdmi_remote_power_checkbox">
        <property name="toolTip">
         <string>Toggles the HDMI-CEC Remote Power control feature, this allows devices connected through HDMI ports to be turned ON/OFF.</string>
        </property>
        <property name="whatsThis">
         <string>Toggles the HDMI-CEC Remote Power control feature, this allows devices connected through HDMI ports to be turned ON/OFF.</string>
        </property>
        <property name="text">
         <string>HDMI-CEC Remote Power</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
